console/console-init/ui/src/modules/address/dialogs/EditAddress/EditAddress.tsx

```diff
@@ -328,6 +328,11 @@ export const EditAddress: React.FunctionComponent = () => {
                 ))}
               </FormSelect>
             </FormGroup>
+          </>
+        )}
+        {(address.type?.trim() === "topic" ||
```

Comment:
Please use enum type `AddressTypes` for values "queue", "topic" and "subscription"